api/CSSStyleProperties.json

```diff
@@ -9,9 +9,16 @@
           },
           "chrome_android": "mirror",
           "edge": "mirror",
-          "firefox": {
-            "version_added": "144"
-          },
+          "firefox": [
+            {
+              "version_added": "144"
+            },
+            {
+              "alternative_name": "CSS2Properties",
+              "version_added": "≤2",
+              "version_removed": "144"
```

Comment:
It looks like `CSS2Properties` was supported since Firefox 1, see: https://github.com/caugner/firefox-releases/blame/v2/dom/public/idl/css/nsIDOMCSS2Properties.idl#L43

```suggestion
              "alternative_name": "CSS2Properties",
              "version_added": "1",
              "version_removed": "144"
```

Reply:
Proof:
<img width="455" height="422" alt="image" src="https://github.com/user-attachments/assets/42738397-ecb0-4b01-b4a9-64a9de506ff7" />